docs/get-started/getting-started-gcp.md

```diff
@@ -196,19 +190,21 @@ gcloud dataproc clusters create $CLUSTER_NAME  \
     --subnet=default 
 ```
 
-The new cluster should be up and running within 3-4 minutes!
+There are no initiailization actions that need to be configured because NVIDIA drivers and Spark RAPIDS resources are already installed in the custom image. The new cluster should be up and running within 3-4 minutes!
```

Comment:
Line wrap, typos, and RAPIDS Accelerator instead of Spark RAPIDS
```suggestion
There are no initialization actions that need to be configured because NVIDIA drivers and
RAPIDS Accelerator resources are already installed in the custom image. The new cluster
should be up and running within 3-4 minutes!
```